lessons/Early-Math/translating/langs/en-us/pages/translation-table1.adoc

```diff
@@ -0,0 +1,22 @@
+= Translation Table (Part 1)
+
+++++
+<style>
+table {grid-auto-rows: 1fr;}
+</style>
+++++
+
+
+Fill in any missing spaces on the table below.
+
+[.FillVerticalSpace, cols="^.^1a,^.^4a,^.^4a,^.^4a", stripes="none", options="header"]
+|===
+| 	 | Math 								| Circle of Evaluation				| Words
+| @n | @ifsoln{@math{5 + 3}}				| @showsoln{(coe '(+ 5 3))}			| Find the sum of 5 and 3
+| @n | @math{(5 + 3) \times 2}				| @showsoln{(coe '(* (+ 5 3) 2))}	| Find the sum of 5 and 3, then double it.
```

Comment:
Are you sure this is more aligned to materials students will see than "twice the sum of 5 and 3" or "double the sum of 5 and 3"?

Perhaps that materials I've seen are outdated, but I am unaccustomed to test and textbook questions splitting language that is to be translated into multiple sentences, as seems to be the norm on this entire page. (Parsing the order from your wording is a much simpler task... perhaps that's where math materials are headed?)

Reply:
Okay, your question prompted me to do some research.  The Common Core standards clearly use phrases like the ones I've included.  I then went and looked at practice LEAP tests to see what sort of phrases were used there. In fifth grade, the relevant question uses both forms. In sixth grade, the relevant question uses the phrasing that you are more familiar with.

And so, I'm not sure. I suppose maybe I should include a mix? Or scaffold them, assuming that it's easier when the wording is given in direction form?

[Here are the results of my research, if you're interested.](https://docs.google.com/document/d/1yrdJxf1DbE_U5i6l5nJd1zfURuDNtE5TTVpOUdzCxBk/edit?usp=sharing)